### PR TITLE
Have added one question to HTML quiz set. starting from line 1529.

### PR DESCRIPTION
**Pull request by hmmalic**
Checklist:

- [ x ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x ] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

**Comment by hmmalic** [comment]
I have added HTML questions to html quiz data , the second commit for correcting a syntax mistake by adding curly bracket at the end.

File: src/data/html-quiz.ts [hunk 1]
```diff
@@ -1525,5 +1525,18 @@ const htmlQuiz = [
...
+  {
+    Question:
+      "In HTML, responsive text size can be set with a" "vw" "unit, which means the",
+    Answer: "<viewport width>",
+    Distractor1: "<versatile width>",
+    Distractor2: "<viewing wide>",
+    Distractor3: "<view length>",
+    Explanation:
+      "The text size can be set with a vw unit, which means the viewport width.Viewport is the browser window size.Example 1vw = 1% of viewport width"
+         Link: "https://www.w3schools.com/html/html_responsive.asp",
```
Comment (jdwilkin4) [inline src/data/html-quiz.ts:1537]: Please fix this syntax error so it can be ready for review

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]

**Comment by hmmalic** [comment]
Hello
I did a second commit by fixing the curly bracket syntax, I have also mentioned in the comment .
Can you please check and let me know .

Thanks
Hina
________________________________

**Comment by jdwilkin4** [comment]
@hmmalic 

Make sure to push up your changes to github so we can see it here on the PR.

File: src/data/html-quiz.ts (hunk 1 above)
Comment (hmmalic) [reply src/data/html-quiz.ts]: I have tried to fix the mistakes can you please check , I am a first time contributor therefore making mistakes .

File: src/data/.vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    "configurations": [
+        {
+            "type": "chrome",
+            "request": "launch",
+            "name": "Launch Chrome against localhost",
+            "url": "http://localhost:8080",
+            "webRoot": "${workspaceFolder}"
+        }
+    ]
+}
```
Comment (jdwilkin4) [inline src/data/.vscode/launch.json:15]: Please remove this .vscode folder because it is not needed.

File: src/data/html-quiz.ts [hunk 3]
```diff
@@ -1525,5 +1525,15 @@ const htmlQuiz = [
...
   },
+  {
+  Question:
+  "In HTML, responsive text size can be set with a vw unit, which means the",
+   Answer: "<viewport width>",
+   Distractor1: "<versatile width>",
+   Distractor2: "<viewing wide>",
+   Distractor3: "<view length>",
+   Explanation: "The full form of VW is viewport width. It works like the percentage unit. Specifying 10vw is equivalent to occupying 10% of entire visible screen width.",
+   Link: "https://www.freecodecamp.org/news/learn-css-units-em-rem-vh-vw-with-code-examples/#why-learn-css-relative-units"
```
Comment (jdwilkin4) [inline src/data/html-quiz.ts:1536]: We already have a quiz question for viewport width in the CSS file. 
Please come up with another question.

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by jdwilkin4** [comment]
Going to close this PR due it inactivity.

If you would like to contribute in the future, please create a new PR 👍

**Pull request closed without merging**